src/test/java/core/basesyntax/service/RegistrationServiceImplTest.java

```diff
@@ -0,0 +1,85 @@
+package core.basesyntax.service;
+
+import static org.junit.jupiter.api.Assertions.assertThrows;
+
+import core.basesyntax.dao.StorageDaoImpl;
+import core.basesyntax.model.User;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+
+class RegistrationServiceImplTest {
+    private RegistrationService registrationService = new RegistrationServiceImpl();
+    private StorageDaoImpl storageDao = new StorageDaoImpl();
+    private User user = new User();
+    private User user1 = new User();
+    private User user2 = new User();
+
+    @BeforeEach
+    void setUp() {
+        user.setLogin("login");
+        user.setPassword("password");
+        user.setAge(18);
+        user1.setLogin("login1");
+        user1.setPassword("somepassword");
+        user1.setAge(18);
+        user2.setLogin("login1");
+        user2.setPassword("anotherpassword");
+        user2.setAge(25);

```

Comment:
You use it only in one test method, so there is no need to do this logic before each